Makefile

```diff
@@ -213,17 +227,76 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
+# $1 - target file
+define generate-crd
+	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > '$1'
+endef
+
+update-crd:
+	$(call generate-crd,pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml)
+.PHONY: update-crd
+
+verify-crd: tmp_file :=$(shell mktemp)
+verify-crd:
+	$(call generate-crd,$(tmp_file))
+	$(diff) '$(tmp_file)' pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml || (echo 'CRD was not regenerated' && false)
+.PHONY: verify-crd
+
+# $1 - target dir
+# $2 - source dir
+define copy-amend-manifests
```

Comment:
nit: this isn't really a generic macro as the name make it sound, `generate-dev-manifests` might be a more descriptive name
the input could be fixes as well